test/SILGen/internal_protocol_refines_public_protocol_with_public_default_implementation.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %target-swift-emit-silgen -verify %s
+
+public protocol A {
+    @_borrowed
+    subscript() -> Int { get }
+}
+
+protocol B: A { }
+
+extension B {
+    public subscript() -> Int { return 0 }
+}
+
+public struct S: B {
```

Comment:
Can you add a CHECK: line that the witness for A.subscript is public?

Reply:
It looks like it's shared visibility. Seems like it shouldn't have to be public, because the witness table is what's public.